src/System.Diagnostics.Tracing/ref/System.Diagnostics.Tracing.cs

```diff
@@ -103,6 +103,7 @@ public enum EventKeywords : long
         Sqm = (long)2251799813685248,
         WdiContext = (long)562949953421312,
         WdiDiagnostic = (long)1125899906842624,
+        MicrosoftTelemetry = (long)562949953421312,
```

Comment:
Simple is fine :)  Probably don't need any for enums, but for new methods, properties, etc., we should have basic verification.